app/src/main/res/layout/activity_help_parameters.xml

```diff
@@ -155,7 +175,7 @@
         android:layout_height="wrap_content"
         android:layout_marginStart="44dp"
```

Comment:
I think we can keep the phone on red. 
Since it is the most important action to do un case of emergency it should pop out.

Reply:
It was initially in green actually haha, but sure :)